src/pages/contact.md

```diff
@@ -11,7 +11,7 @@ There are various channels where you can reach us.
 
 - We welcome your **bug** reports and **feature requests** through our community channels mentioned above.
 
-- For **security-related issues**, review our [Security notices](../../docs/reference/notices) for the most up-to-date information on known issues and steps to report a vulnerability so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
+- For **security-related issues**, review our [Security notices](../docs/reference/notices) for the most up-to-date information on known issues and steps to report a vulnerability so we can solve the problem as quickly as possible. Do not use GitHub for security-related issues.
```

Comment:
I actually think this URL is incorrect in production, too...but it works out fine, because if you go two up from one level deep, you still just end up at the root. 

This will be fixed in `main`, not in the archive branches.

Reply:
How are you finding these links? Are they appearing as build issues or through a search?